package/README.md

```diff
@@ -71,6 +71,17 @@ You can then use that `getClient` function in your server components:
 const { data } = await getClient().query({ query: userQuery });
 ```
 
+#### Multiple clients
+
+If you need to manage multiple clients, for example if you have different endpoints for content previews,
```

Comment:
To be honest, the "for example" feels a little too specific. I'm not sure most apps would have a "content preview", nor do I think many people might understand what this is (to be honest I'm one of those 😅).

Instead, could we use either something like an auth token, or a versioned endpoint or something as an example? This might resonate with readers a bit more. I'm also for removing this part of the sentence altogether if we can't come up with a great example.